community/releases/next-release-preview/index.md

```diff
@@ -12,3 +12,18 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.26
+
+### Git pull support for git/repo artifacts
+
+In Spinnaker < 1.26 every time a `git/repo` artifact was needed during a pipeline execution, clouddriver cloned the repository, returned the files and deleted the clone immediately.
+
+Now in Spinnaker 1.26 we added support for "caching" git repositories, so only the first time the repository is needed it will be cloned, and subsequent times clouddriver will do a `git pull` to only download updates. It is expected that this dramatically improves execution times and reliability when working with big repositories.
+
+This is an op-in feature that is disabled by default, to use it you need to add to clouddriver profile configuration the following options:
+
+```yaml
+artifacts:
+  gitrepo:
+    clone-retention-minutes: 0           # (Default: 0). How much time to keep clones. 0: no retention, -1: retain forever
+    clone-retention-max-bytes: 104857600 # (Default: 100 MB). Maximum amount of disk space to use for clones.
```

Comment:
What happens when the size of clones hits this limit?

Reply:
When a request comes in and it detects that the size limit has been reached, it will delete the clone after returning. I'll add that to the description.

Reply:
Sounds good.  Thanks.